Components/avatar/avatar.css

```diff
@@ -0,0 +1,36 @@
+.avatar{
+    --size:50px;
+    height: var(--size);
+    width: var(--size);
+    border-radius: 100%;
+
+    color: white;
+    background:grey;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+}
+
+.avatar img{
+    height: inherit;
+    width: inherit;
+    border-radius: 100%;
```

Comment:
why 100%? 50% is enough if you are aiming for circle

Reply:
Thanks, Rohit! I always used to do 100%